nims-notes-things/src/Form.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
```

Comment:
Try to get prettier installed with your vscode... will help with formatting and then you don't have to worry about indentation or semi-colons 
https://www.codereadability.com/automated-code-formatting-with-prettier/#:~:text=Open%20the%20Command%20Palette%20(under,once%20the%20installation%20is%20complete